modules/nixos/services/bobaboard/default.nix

```diff
@@ -87,20 +114,63 @@ in
         Type = "oneshot";
         User = "bobaboard";
         Group = "bobaboard";
-        ExecStart = "${bobabackend-packages.bobadatabase}/bin/bobadatabase";
+        # ExecStart = "${bobabackend-packages.bobadatabase}/bin/bobadatabase";
         
         # Uncomment this to have the script re-run when boba-server restarts
         # RemainAfterExit = "yes";
       };
 
       # Only run the script if the .migrate file is not present
       # Remove ExecStart when enabling this
-      # script = ''
-      #   if ! [ -f /var/lib/bobaboard/.migrate ]; then
-      #     touch /var/lib/bobaboard/.migrate
-      #     ${bobabackend-packages.bobadatabase}/bin/bobadatabase
-      #   fi
-      # '';
+      script = ''
+        if ! [ -f /var/lib/bobaboard/.migrate ]; then
+          touch /var/lib/bobaboard/.migrate
+          ${bobabackend-packages.bobadatabase}/bin/bobadatabase
+        fi
+      '';
+    };
+
+    security.acme.acceptTerms = true;
+    security.acme.defaults.email = "essential.randomn3ss@gmail.com";
```

Comment:
These should be user configured, not a part of the module. We can move these to our system config instead.